SiemensIXBlazor/Components/NavigationMenu/NavigationMenuCategory.razor

```diff
@@ -1,5 +1,4 @@
-﻿@using SiemensIXBlazor.Components;
```

Comment:
Other usings are inside the razor pages. Is there any advantage to move this inside the class?

Reply:
no particular reason, works both way. I can put it back

Reply:
Yes, please. It's important for consistency.

Reply:
done